src/processwatch.h

```diff
@@ -205,8 +202,8 @@ extern struct pw_opts_t pw_opts;
 
 /* Reading from BPF and storing the results */
 #include "results.h"
-#ifndef CAPSTONE
```

Comment:
this breaks the compilation on aarch64...it doesn't like the asm in the get_vendor. This was ifndef'ed out (although arguably I should have used the ARM not CAPSTONE define)...but now it's getting compiled in

Reply:
Ah, I'll have to fix `is_amd_arch()` properly, then. That was recently added. We should first check for ARM/x86, _then_ only run `is_amd_arch()` if it's x86.

Reply:
Can we not just use the `__aarch64__`, `__arm__`, and `__x86_64__` macros to identify the architecture, instead of defining `ARM` in `build.sh`?

Reply:
If we _can_, then I can just call `is_amd_arch()` iff `__x86_64__` is defined, and replace `ARM` with `__aarch64__` everywhere.

Reply:
Yeah that's a better idea

> Can we not just use the `__aarch64__`, `__arm__`, and `__x86_64__` macros to identify the architecture, instead of defining `ARM` in `build.sh`?

yeah that's a better idea